web/modules/custom/joinup_rdf/src/RequirementsHelper.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\joinup_rdf;
+
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\Core\StringTranslation\TranslationManager;
+use Drupal\sparql_entity_storage\Database\Driver\sparql\ConnectionInterface;
+
+/**
+ * Implements helper methods related to the requirements.
+ */
+class RequirementsHelper {
+
+  use StringTranslationTrait;
+
+  /**
+   * The SQL connection class for the SPARQL database storage.
+   *
+   * @var \Drupal\sparql_entity_storage\Database\Driver\sparql\ConnectionInterface
+   */
+  protected $sparql;
+
+  /**
+   * The translation manager service.
+   *
+   * @var \Drupal\Core\StringTranslation\TranslationManager
+   */
+  protected $translation;
+
+  /**
+   * Constructs a new RequirementsHelper.
+   *
+   * @param \Drupal\sparql_entity_storage\Database\Driver\sparql\ConnectionInterface $sparql
+   *   The SQL connection class for the SPARQL database storage.
+   * @param \Drupal\Core\StringTranslation\TranslationManager $translation
+   *   The translation manager service.
+   */
+  public function __construct(ConnectionInterface $sparql, TranslationManager $translation) {
+    $this->sparql = $sparql;
+    $this->translation = $translation;
+  }
+
+  /**
+   * Returns a list of error messages for the joinup_rdf requirements.
+   *
+   * @return array
+   *   An array of string messages.
+   */
+  public function getRequirementErrors() {
```

Comment:
We should say that we're returning the list of error messages as a render array. Also the return type is still `: array`